integration-tests/typings/embrace.d.ts

```diff
@@ -27,17 +27,64 @@ interface EmbraceSpanData {
   attributes: EmbraceSpanAttribute[];
 }
 
+interface EmbraceLogData {
```

Comment:
should remove the span specific stuff here, e.g. `trace_id`, `status`, `events`. For attributes let's rename `EmbraceSpanAttribute` -> `EmbraceAttribute` since it's used for both